analyses/independent-samples/02-generate-independent-rnaseq.R

```diff
@@ -83,7 +83,7 @@ independent_rna_primary_plus_polya <- sample_df %>%
                           histology_df = .,
                           match_type = "independent_dna_plus_only_rna",
                           tumor_description_rna_only = "primary_plus",seed = 2020) %>%
-  write_tsv(rnaseq_primplus_polya_file)
+  readr::write_tsv(rnaseq_primplus_polya_file)
```

Comment:
```suggestion
  readr::write_tsv(rnaseq_primplus_polya_file)
```

I think now is the time to combine the polya and stranded samples into one file  with our expression data being combined. Especially since there are duplicate samples which have been stranded seq and polya seq and they may exist in both lists, making them non-independent. - can you make an update to this?